tools/ffi_coverage/src/allowlist.rs

```diff
@@ -213,14 +213,14 @@ lazy_static::lazy_static! {
         "icu::properties::maps::CodePointMapData::from_code_point_trie",
         "icu::properties::maps::CodePointMapData::to_code_point_trie",
         "icu::properties::maps::CodePointMapDataBorrowed::iter_ranges",
-        "icu::properties::sets::CodePointSetData::as_code_point_inversion_list",
-        "icu::properties::sets::CodePointSetData::from_code_point_inversion_list",
-        "icu::properties::sets::CodePointSetData::to_code_point_inversion_list",
         "icu::properties::sets::UnicodeSetData::as_code_point_inversion_list_string_list",
         "icu::properties::sets::UnicodeSetData::from_code_point_inversion_list_string_list",
         "icu::properties::sets::UnicodeSetData::to_code_point_inversion_list_string_list",
         "icu::properties::script::ScriptWithExtensionsBorrowed::get_script_extensions_set", // returns an inversion list
```

Comment:
we seem to have inversion lists on the API, why is this allowlisted? could you revise the comments on which types are exposed and which are not?

Reply:
I'd rather not do that here, we have an open issue filed about homogenizing this file and our strategy here. It's very messy right now and a lot of things there ought to be moved into rust_link.

Reply:
That said I could add that particular API